heka/files/lua/common/afd.lua

```diff
@@ -128,9 +128,18 @@ function reset_alarms()
     alarms = {}
 end
 
+local disallowed_dimensions = {
+    name = true,
+    value = true,
+    hostname = true,
+    member = true,
+    no_alerting = true,
+    tag_fields = true,
+}
```

Comment:
Right, I just wanted to list the disallowed dimensions in the code that does deal with the message fields.